src/main/kotlin/com/openlattice/shuttle/destinations/PostgresDestination.kt

```diff
@@ -113,6 +111,7 @@ class PostgresDestination(
                                 esSw.elapsed(TimeUnit.MILLISECONDS)
                         )
                         val esCount = entities.groupBy { getPartition(it.first, partitions) }
+                                .toSortedMap()
```

Comment:
I guess we do this in a bunch of places, but since there's already so much contention for these connections I feel like we should be doing as minimal work as possible inside them, so ideally grouping/sorting the entities map before the connection is acquired. (but that can be for another day)